src/stores/notifications.ts

```diff
@@ -0,0 +1,49 @@
+import { defineStore } from 'pinia'
+
+export const useNotificationsStore = defineStore('notifications', {
+  state: () => {
+    return {
+      notifications: {
```

Comment:
Some basic typing would be nice here: `Record<string, {name: string, isEnabled: boolean}>`

Reply:
We're not passing a recort to updatefunction, we're passing an entire object